frontend/src/components/Handler/ResponseHandler.tsx

```diff
@@ -0,0 +1,53 @@
+import React from 'react'
```

Comment:
This component looks fine to me but here i would suggest `notistack` lib, it basically shows snackbar and has some cool transitions as well. @devendra0009 It would be great if you could explore and add that here in this fix. 
cc @DevilsAutumn

Reply:
Also, just to add, we will avoid showing a dialog this big in case of errors/API fails. It is a bad UX practice.

Reply:
sure can implement that

Reply:
> Also, just to add, we will avoid showing a dialog this big in case of errors/API fails. It is a bad UX practice.

Yeah, Probably a small box on top right corner of the page would be better.